eng/config.json

```diff
@@ -72,6 +72,15 @@
             "Name": "keyvault/azsecrets",
             "CoverageGoal": 0.86
         },
+        {
+            "Name": "data/aztables",
+            "CoverageGoal": 0.77,
+            "EnvironmentVariables": {
+                "TABLES_STORAGE_ACCOUNT_NAME": "fakeaccount",
+                "TABLES_PRIMARY_STORAGE_ACCOUNT_KEY": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
+                "TABLES_SHARED_ACCESS_SIGNATURE": "?sig=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
```

Comment:
Are these values going to get flagged by cred scanners?

Reply:
It's not new, they were part of the previous 'data' block.

Reply:
Can we use a more obviously fake value everywhere?

Reply:
Ah, these are Azurite (ie, the emulator) connection values. So they're not secret at all.